include/rewd_controllers/JointTrajectoryControllerBase.hpp

```diff
@@ -19,6 +20,38 @@
 
 namespace rewd_controllers
 {
+
+/// The JointTrajectoryControllerBase uses a bunch of ros parameters as configuration
+/// See this full example configuration in yaml format:
+/// 
+/// rewd_trajectory_controller:
```

Comment:
I think it'd suffice to just mention that we support `stopped_velocity_tolerance`  and `goal` for constrains.